src/main/java/org/takes/rq/RqHeaders.java

```diff
@@ -89,36 +88,32 @@
         public Base(final Request req) {
             super(req);
         }
+
+        /**
+         * @todo #490 We need to implement a VerboseList and its tests
+         *  like VerboseIterable. It should provide means to throw an exception
+         *  with a given error message when it otherwise throws an
+         *  IndexOutOfBoundsException.
+         */
         @Override
-        public Iterable<String> header(final CharSequence key)
+        public List<String> header(final CharSequence key)
             throws IOException {
-            final Map<String, List<String>> map = this.map();
-            final List<String> values = map.get(
+            final List<String> values = this.map().get(
                 key.toString().toLowerCase(Locale.ENGLISH)
             );
-            final Iterable<String> iter;
+            // @todo #490 This list must be wrapped in a VerboseList
```

Comment:
@aliceice please remove this todo block ant put the text into todo above
we don't need two todos on the same ticket